packages/ckb-types/index.d.ts

```diff
@@ -10,6 +10,7 @@ declare namespace CKBComponents {
   export type Version = UInt32
   export type Difficulty = bigint
   export type BlockNumber = string
+  export type EpochInHeader = string
```

Comment:
How about `EpochNumber` instead?

Reply:
> How about `EpochNumber` instead?

A little confusing, seems to be the type of number

Reply:
Exactly because `epoch` in header is just a number 😿

Reply:
> Exactly because `epoch` in header is just a number 😿

Maybe it will be an object in the future 😿